examples/catalog-module/index.js

```diff
@@ -11,7 +11,7 @@ ReactDOM.render(
         path: '/',
         title: 'Foo',
         imports: {Foo: require('./components/Foo/Foo')},
-        component: require('./components/Foo/Foo.docs.md')
+        component: ContentLoader('./components/Foo/Foo.docs.md')
```

Comment:
D'oh, yes of course! Stupid me 😅 

But (just to be a pain in the ass), the behavior is still not the same 😃  But nevermind!